dev/com.ibm.ws.microprofile.config.1.4_fat/fat/src/com/ibm/ws/microprofile/config14/test/apps/badobserver/BadObserverBean.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.config14.test.apps.badobserver;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Dependent
+public class BadObserverBean {
+
+    @Inject
+    @ConfigProperty(defaultValue = "hello")
+    String property;
+
+    public void dummyTest() {
+        //no-op
+    }
+}
```

Comment:
It's not obvious why this class is here, doesn't appear to be needed.

Additionally, is `hello` actually defined somewhere?

Reply:
This is just a dummy bean to ensure that CDI is enabled and that the ApplicationScope is active.... and can therefore be observed. It is a little belt and braces.